python/ray/rllib/agents/trainer.py

```diff
@@ -402,6 +402,8 @@ def _setup(self, config):
 
         # Merge the supplied config with the class default
         merged_config = copy.deepcopy(self._default_config)
+        if self.config["evaluation_interval"]:
```

Comment:
If eval_config = {} via COMMON_CONFIGS, than by default deep update sees content of non-empty eval_config dict as illegal configs.